frontera/contrib/backends/sqlalchemy/__init__.py

```diff
@@ -141,41 +142,38 @@ def get_next_requests(self, max_next_requests, **kwargs):
         return next_pages
 
     def page_crawled(self, response, links):
-        db_page, _ = self._get_or_create_db_page(response)
```

Comment:
See https://github.com/scrapinghub/frontera/pull/70#issuecomment-141939767, unfortunately without solving canonical urls that code will end up loosing all redirected urls.